src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.url = url + slug;
```

Comment:
You already have `url` as a property in the parent `PageObject` class. Instead of redefining it in the `ArticlePage` class, you can just modify the existing `url` property. This can be done by calling `super(url + slug)` in the constructor of the `ArticlePage` class.